tests/nn/data_parallel/test_fsdp_grad_acc.py

```diff
@@ -17,11 +18,15 @@
 from .test_fsdp import DistributedTest, NestedWrappedModule, rename_test, spawn_and_init
 
 
-class TestNoSync(DistributedTest):
+class TestGradAcc(DistributedTest):
```

Comment:
<3 naming

Reply:
I changed the name since this is test both no_sync and without no_sync now. In both cases, it is testing grad acc. So I named it TestGradAcc. Let me know or just put in a github suggestion for a better name.